packages/cf-style-const/src/variables.js

```diff
@@ -4,7 +4,18 @@ const colorOffsetLight = 0.15;
 const colorOffsetDark = 0.07;
 
 export default {
+  gradient: {
```

Comment:
If we're providing gradients, let's provide them in a usable format (e.g. `linear-gradient` syntax). Otherwise users can use these in a way that's against design's intent (e.g. `radial-gradient(ellipse farthest-corner at 45px 45px , gradient.primary[0] 0%, gradient.primary[1] 50%)`).